x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/components/analytics_tables/analytics_table.test.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { mountWithIntl, mockKibanaValues } from '../../../../../__mocks__';
+import '../../../../__mocks__/engine_logic.mock';
+
+import React from 'react';
+import { EuiBasicTable, EuiBadge, EuiEmptyPrompt } from '@elastic/eui';
+
+import { AnalyticsTable } from './';
+
+describe('AnalyticsTable', () => {
+  const { navigateToUrl } = mockKibanaValues;
+
+  const items = [
+    {
+      key: 'some search',
+      tags: ['tagA'],
+      searches: { doc_count: 100 },
+      clicks: { doc_count: 10 },
+    },
+    {
+      key: 'another search',
+      tags: ['tagB'],
+      searches: { doc_count: 99 },
+      clicks: { doc_count: 9 },
+    },
+    {
+      key: '',
+      tags: ['tagA', 'tagB'],
+      searches: { doc_count: 1 },
+      clicks: { doc_count: 0 },
+    },
+  ];
+
+  it('renders', () => {
+    const wrapper = mountWithIntl(<AnalyticsTable items={items} />);
```

Comment:
Can you remind me, when do we use `mountWithIntl` over regular old `shallow` or `mount`?

Reply:
In this case it's because:

1. I actually want to fully render the entire table (`mount`)
2. Because we're using `<FormattedDate />` or `<FormattedMessage />` etc. which requires a react-intl wrapper component.

Reply:
> Because we're using <FormattedDate /> or <FormattedMessage /> etc. which requires a react-intl wrapper component.

Ah, OK, I didn't realize that.